System.IO.FileSystem/Directory.cs

```diff
@@ -3,151 +3,309 @@
 // See LICENSE file in the project root for full license information.
 //
 
+using System.Collections;
 using System.Runtime.CompilerServices;
 
 namespace System.IO
 {
     /// <summary>
-    /// Class for managing directories
+    /// Exposes static methods for creating, moving, and enumerating through directories and subdirectories. This class cannot be inherited.
     /// </summary>
     public static class Directory
     {
-        #region Static Methods
-
         /// <summary>
         /// Determines a list of available logical drives.
         /// </summary>
         /// <returns>String[] of available drives, ex. "D:\\"</returns>
         [Obsolete("Use DriveInfo.GetDrives() instead.")]
         public static string[] GetLogicalDrives()
         {
-            return GetLogicalDrivesNative();
+            throw new NotImplementedException();

```

Comment:
why throw? If marked as obsolete, I would keep it. Otherwise, if it throws, it's not usefull as it won't break the code. Better then remove it fully! At least, it forces to change the code rather than having something throwing without any perceived reason.

Reply:
Agreed. Actually that was my thought process too, just lost it when switching context... 🤪